client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -925,6 +972,16 @@ export default {
       return isSorted ? { color: 'black' } : {}
     },
 
+    handleShowDeleted (showDeleted = true) {
```

Comment:
this method can literally be done in 2 lines, and as said above it doesn't need the pram since you know what mode you're currently in